code/ui/manager/src/components/notifications/NotificationList.tsx

```diff
@@ -4,41 +4,48 @@ import type { State } from '@storybook/manager-api';
 import { styled } from '@storybook/theming';
 import type { CSSObject } from '@storybook/theming';
 import NotificationItem from './NotificationItem';
+import { MEDIA_DESKTOP_BREAKPOINT } from '../../constants';
 
-const List = styled.div<{ placement?: CSSObject }>(
-  {
-    zIndex: 10,
+interface NotificationListProps {
+  notifications: State['notifications'];
+  clearNotification: (id: string) => void;
+}
+
+export const NotificationList: FC<NotificationListProps> = ({
+  notifications,
+  clearNotification,
+}) => {
+  return (
+    <List>
+      {notifications &&
+        notifications.map((notification) => (
+          <NotificationItem
+            key={notification.id}
+            onDismissNotification={(id: string) => clearNotification(id)}
+            notification={notification}
+          />
+        ))}
+    </List>
+  );
+};
+
+const List = styled.div<{ placement?: CSSObject }>({
+  zIndex: 200,
+  position: 'fixed',
+  left: 20,
+  bottom: 60,
+  maxHeight: 'calc(100vh - 40px)',
+  overflowY: 'scroll',
```

Comment:
@cdedreuille  Check out the story, especially in dark mode the scroll bar is clearly visible, do you see the same thing?

https://635781f3500dd2c49e189caf-wamswpsigz.chromatic.com/?path=/story/manager-notifications-notificationlist--single&globals=theme:dark

![image](https://github.com/storybookjs/storybook/assets/5678122/77a9b841-c2cf-4e46-831c-39c5d72019d6)

Reply:
Ah damn. What browser are you testing on?

Reply:
I'm not even sure we need that to be honest. I was just trying to make sure you can scroll even if you have a lot of notifications but if you do, this means that you can't use the sidebar :)

Reply:
I might just remove this overflow

Reply:
Done